src/serac/physics/functional_residual.hpp

```diff
@@ -83,8 +83,9 @@ class FunctionalResidual<spatial_dim, OutputSpace, Parameters<InputSpaces...>,
    * // DependsOn<active_parameters...> can be indices into fields which the body integral may depend on
    * @param body_name The name of the registered domain over which the body force is applied. If nothing is supplied
    * the entire domain is
-   * @param body_integral A function describing the body force applied
-   * used.
+   * @param body_integral A function describing the body force applied.  Our convention for the sign of the residual
+   * vector is to negate the body force internally so the resulting residual is a 'negative force'.  This is to ensure
+   * that the Jacobian of the residual is positive definite for most physics.
```

Comment:
I find this documentation confusing, but I admit it's hard to articulate. I think of this as documentation for people trying to use a physics module. They need to look up what the sign convention is, but the reason for the convention is an implementation detail they don't need to know. 

```suggestion
   * @param body_integral A function returning a volumetric source. Expected sign convention is that
   * the source would do positive work through a positive change in the conjugate generalized
   * displacement.
```

Reply:
As an aside, the variable name `body_integral` is misleading. It's more accurately the `body_integrand`. We can deal with that later.